### PR TITLE
Documents Unsupported ansible modules Q/A

### PR DESCRIPTION
**Pull request by flickerfly**
Adds a question and the answer to the documentation for dealing with dependencies for ansible modules that aren't included in the ansible-operator image.

**Description of the change:**
Adds a question and the answer to the documentation for dealing with dependencies for ansible modules that aren't included in the ansible-operator image.

**Motivation for the change:**
I had to piece this together myself and thought it would be good to document for others (and when I forget).

### CODE REVIEW

**Comment by flickerfly** [comment]
Thanks to @camilamacedo86 for the encouragement to just do it.

**Comment by fabianvf** [comment]
@flickerfly yeah there's some issues with the test infrastructure right now, don't worry about it

File: doc/faq.md [hunk 1]
```diff
@@ -36,6 +36,19 @@ Add the following to your `deploy/role.yaml` file to grant the operator permissi
...
+Unfortunately, adding the entire dependency tree for all Ansible modules would be excessive. Fortunately, you can add it easily. Simply edit your build/Dockerfile. You'll want to change to root for the install command, just be sure to swap back using a series of commands like the following right after the `FROM` line.
+
+```
+USER 0
+RUN yum -y install my-dependency
+RUN pip3 install my-python-dependency
+USER 1001
+```
+
+If you aren't sure what dependencies are required, start up a container using the image in the `FROM` line as root. That's probably look something like this.
```
Comment (fabianvf) [inline doc/faq.md]:
```suggestion
If you aren't sure what dependencies are required, start up a container using the image in the `FROM` line as root. That will look something like this.
```

**Pull request merged**